source/instructors/setupcourses/assignment-settings/assignment-duration.rst

```diff
@@ -23,7 +23,7 @@ To set the assignment duration, follow these steps:
    - Click the **Calendar** icon and select the date.
    - Click the **Clock** icon and select the time.
 
-   .. Note:: The time zone is automatically determined by your browser time zone.
+   .. Note:: The time zone is automatically from the Course time zone but if you wish to change to a different time zone, you can do so clicking the 'click here' link that will take you to the **Course Details** area where you can change the timezone for all the assignments in the course.
```

Comment:
Slight wording change line 26  the words after Note:

The time zone defaults to the Course time zone, if you wish to change too a different..